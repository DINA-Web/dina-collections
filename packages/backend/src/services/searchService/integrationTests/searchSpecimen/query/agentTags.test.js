const {
  describe: apiSampleDescribe,
  hook,
} = require('common/src/testUtilities/envBackendApiSampleData')
const waitForApiRestart = require('../../../../../utilities/test/waitForApiRestart')
const testSpecification = require('./testCases/tags/agentTags')
const runApiTagTests = require('../../utilities/runApiTagTests')

apiSampleDescribe(`searchSpecimen - query - agentTags`, () => {
  hook(beforeAll, () => {
    return waitForApiRestart()
  })
  runApiTagTests(testSpecification)
})
