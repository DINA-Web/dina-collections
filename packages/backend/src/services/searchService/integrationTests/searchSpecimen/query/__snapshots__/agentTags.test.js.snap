// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`searchSpecimen - query - agentTags tagTypeAggregation raw returns all tag types matches snapshot 1`] = `
Array [
  Object {
    "attributes": Object {
      "count": 32,
      "key": "record history by",
      "tagType": "record history by",
      "type": "agentTag",
    },
    "id": "record history by",
    "type": "customObject",
  },
  Object {
    "attributes": Object {
      "count": 15,
      "key": "collector",
      "tagType": "collector",
      "type": "agentTag",
    },
    "id": "collector",
    "type": "customObject",
  },
  Object {
    "attributes": Object {
      "count": 9,
      "key": "determined by",
      "tagType": "determined by",
      "type": "agentTag",
    },
    "id": "determined by",
    "type": "customObject",
  },
  Object {
    "attributes": Object {
      "count": 7,
      "key": "catalog card created by",
      "tagType": "catalog card created by",
      "type": "agentTag",
    },
    "id": "catalog card created by",
    "type": "customObject",
  },
  Object {
    "attributes": Object {
      "count": 5,
      "key": "handed in by",
      "tagType": "handed in by",
      "type": "agentTag",
    },
    "id": "handed in by",
    "type": "customObject",
  },
]
`;
